apisix/schema_def.lua

```diff
@@ -620,7 +620,7 @@ _M.consumer = {
     type = "object",
     properties = {
         username = {
-            type = "string", minLength = 1, maxLength = 32,
+            type = "string", minLength = 1, maxLength = 100,
```

Comment:
```suggestion
            type = "string", minLength = 1, maxLength = rule_name_def.maxLength,
```

@spacewander Do you mean referencing this field separately?